__tests__/app.test.js

```diff
@@ -409,3 +409,35 @@ describe('Patch votes by Article ID', () => {
         })
     })
 
+    describe('Delete comments by comment_id', () => {
+        describe('DELETE/api/comments/:comment_id', () => {
+            it('should return a status of 204', () => {
+                return request(app).delete('/api/comments/1').expect(204);
+            })
+            it('should return no content', () => {
+                return request(app).delete('/api/comments/1').expect(204).then(({body}) => {
+                    expect(body).toEqual({})
+                })
+            })
+            it('should delete the given comment from the comments table', () => {
+                return request(app).delete('/api/comments/1').expect(204).then(() => {
+                    return request(app).get('/api/articles/9/comments').then(({body}) => {
+                        const comments = body;
```

Comment:
Nice while this will work it would be a better test to iterate through the comments to make sure the one you deleted doesn't exist.